test/generic/DirectSum-test.jl

```diff
@@ -67,3 +73,18 @@ end
       end
    end
 end
+
+@testset "Generic.DirectSum.more maps..." begin
```

Comment:
Isn't there a more descriptive name we can give to this test set? Also, we don't put spaces in the names, but underscores. Or can it be combined with the existing testset for DirectSum maps?